### PR TITLE
update ffmpeg version

### PR DESCRIPTION
**Pull request by Artiume**

### CODE REVIEW

File: general/administration/installing.md [hunk 1]
```diff
@@ -240,11 +240,11 @@ Create four sub-directories for Jellyfin data.
 sudo mkdir data cache config log
 ```
 
-If you are running Debian or a derivative, you can also download and install an ffmpeg release built specifically for Jellyfin. Be sure to download the latest release that matches your OS (4.2.1-3 for Debian Stretch assumed below).
+If you are running Debian or a derivative, you can also download and install an ffmpeg release built specifically for Jellyfin. Be sure to download the latest release that matches your OS (4.2.1-4 for Debian Stretch assumed below).
```
Comment (JustAMan) [inline general/administration/installing.md]: this is going to be out of date soon, can we make a more stable instruction page that doesn't need updating each time we publish a new package?

Comment (joshuaboniface) [reply general/administration/installing.md]: I'd rather switch this whole line to encourage either (a) jellyfin-ffmpeg for Debuntu, or (b) "The latest 4.1 FFMpeg release from your distribution". No need to go into the -X version numbers or anything.

-4 is already out of date ;-)

Comment (Artiume) [reply general/administration/installing.md]: I added the repo link so it's easier to find the appropriate package.

Comment (Artiume) [reply general/administration/installing.md]: also updated the example to .5 :P

File: general/administration/installing.md [hunk 2]
```diff
@@ -240,11 +240,11 @@ Create four sub-directories for Jellyfin data.
 sudo mkdir data cache config log
 ```
 
-If you are running Debian or a derivative, you can also download and install an ffmpeg release built specifically for Jellyfin. Be sure to download the latest release that matches your OS (4.2.1-4 for Debian Stretch assumed below).
+If you are running Debian or a derivative, you can also [download](https://repo.jellyfin.org/releases/server/debian/versions/jellyfin-ffmpeg/) and install an ffmpeg release built specifically for Jellyfin. Be sure to download the latest release that matches your OS (4.2.1-5 for Debian Stretch assumed below).
```
Comment (JustAMan) [inline general/administration/installing.md]:
```suggestion
If you are running Debian or a derivative, you can also [download](https://repo.jellyfin.org/releases/server/debian/versions/jellyfin-ffmpeg/) and install an ffmpeg release built specifically for Jellyfin. Be sure to download the latest release that matches your OS (4.2.1-5 for Debian Stretch assumed below).
```

**Comment by Artiume** [comment]
Yeah... 😑 Lol

**Pull request merged**